lametro/templates/lametro/index.html

```diff
@@ -11,6 +11,9 @@ <h1 class="home-header">Los Angeles County Metropolitan Transportation Authority
                 <p class="h3">
                     <span class="text-bg">Search Metro Board Reports (2015 – present)</span>
                 </p>
+                <p style="color: #ffffff;font-weight: bold;">
```

Comment:
i'm not sure i like the bold text, but i understand it's tough to read with the background image. what do you think about increasing the opacity of the white in the gradient overlay, in order to fade the image a bit, and using normal font weight here instead?

Reply:
it'd also be preferable not to use inline style, if possible. does `text-bg` do the trick?